openid-4-verifiable-presentations-1_0.md

```diff
@@ -3155,6 +3155,13 @@ This specification registers the following client metadata parameters
 in the IANA "OAuth Dynamic Client Registration Metadata" registry [@IANA.OAuth.Parameters]
 established by [@!RFC7591].
 
+### encrypted_response_enc_values_supported
+
+* Client Metadata Name: `encrypted_response_enc_values_supported`
+* Client Metadata Description: Array of strings, where each string is a JWE [@!RFC7516] `enc` algorithm that can be used as the content encryption algorithm for encrypting the Response
```

Comment:
```suggestion
* Client Metadata Description: An array of strings, where each string is a JWE [@!RFC7516] `enc` algorithm that can be used as the content encryption algorithm for encrypting the Response
```

Reply:
Thanks for the suggestion - I thought about this and decided to leave it as it is for consistency with other entries in https://www.iana.org/assignments/oauth-parameters/oauth-parameters.xhtml#client-metadata which all seem to omit a particle.